packages/web-components/src/components/pricing-table/__stories__/pricing-table.stories.ts

```diff
@@ -0,0 +1,281 @@
+        </dds-pricing-table-cell>
+      `;
+    default:
+      return html`
+        <dds-pricing-table-cell>
+          <dds-pricing-table-cell-annotation>
+            Sed quis neque ultrices, convallis augue non, scelerisque massa.
+          </dds-pricing-table-cell-annotation>
+        </dds-pricing-table-cell>
+      `;
+  }
+};
+
+const renderBodyRow = (
+  columnCount: number,
+  rowNum: number,
+  cellType: CELL_TYPES,
+  rowHeaders: boolean = true
+): TemplateResult => html`
+  <dds-pricing-table-row>
+    ${(() => {
+      const cells: TemplateResult[] = [
+        html`
+          ${rowHeaders
+            ? html`
+                <dds-pricing-table-header-cell scope="row">Row ${rowNum}</dds-pricing-table-header-cell>
+              `
+            : html`
+                <dds-pricing-table-cell
+                  >Row ${rowNum} Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dui magna, finibus id tortor sed,
+                  aliquet bibendum augue. Aenean posuere sem vel euismod dignissim.</dds-pricing-table-cell
+                >
+              `}
+        `,
+      ];
+      for (let i = 1; i < columnCount; i++) {
+        cells.push(renderBodyCell(cellType));
+      }
+      return cells;
+    })()}
+  </dds-pricing-table-row>
+`;
+
+export const Default = ({ parameters }) => {
+  const { colSpan1, colSpan2, colSpan3, colSpan4, highlightCol, highlightLabel, columnCount, heading } =
+    parameters?.props?.PricingTable ?? {};
+  return html`
+    <dds-pricing-table
+      col-span-1="${colSpan1 ?? ''}"
+      col-span-2="${colSpan2 ?? ''}"
+      col-span-3="${colSpan3 ?? ''}"
+      col-span-4="${colSpan4 ?? ''}"
+      highlight-column="${highlightCol}"
+      highlight-label="${highlightLabel}"
+    >
+      ${renderHead(columnCount, heading)}
+      <dds-pricing-table-body>
+        ${renderBodyRow(columnCount, 1, CELL_TYPES.ICON)} ${renderBodyRow(columnCount, 2, CELL_TYPES.EMPTY)}
+        ${renderBodyRow(columnCount, 3, CELL_TYPES.TEXT)}
+      </dds-pricing-table-body>
+    </dds-pricing-table>
+  `;
+};
+
+export const WithoutRowHeaders = ({ parameters }) => {
+  const { colSpan1, colSpan2, colSpan3, colSpan4, highlightCol, highlightLabel, columnCount, heading } =
+    parameters?.props?.PricingTable ?? {};
+  return html`
+    <dds-pricing-table
+      col-span-1="${colSpan1 ?? ''}"
+      col-span-2="${colSpan2 ?? ''}"
+      col-span-3="${colSpan3 ?? ''}"
+      col-span-4="${colSpan4 ?? ''}"
+      highlight-column="${highlightCol}"
+      highlight-label="${highlightLabel}"
+    >
+      ${renderHead(columnCount, heading)}
+      <dds-pricing-table-body>
+        ${renderBodyRow(columnCount, 1, CELL_TYPES.ICON, false)} ${renderBodyRow(columnCount, 2, CELL_TYPES.EMPTY, false)}
+        ${renderBodyRow(columnCount, 3, CELL_TYPES.TEXT, false)}
+      </dds-pricing-table-body>
+    </dds-pricing-table>
+  `;
+};
+
+export const WithSubheaders = ({ parameters }) => {
```

Comment:
We need to title-case these variations in storyboo
<img width="224" alt="Screen Shot 2022-04-05 at 5 46 11 PM" src="https://user-images.githubusercontent.com/25532785/161855228-6a4cc6c7-24a6-4c7c-b84a-f6a3cd9f56eb.png">
k